src/library/scala/math/Ordering.scala

```diff
@@ -422,14 +452,46 @@ object Ordering extends LowPriorityOrderingImplicits {
     }
     implicit object IeeeOrdering extends IeeeOrdering
   }
-  @deprecated("There are multiple ways to order Floats (Ordering.Float.TotalOrdering, " +
-    "Ordering.Float.IeeeOrdering). Specify one by using a local import, assigning an implicit val, or passing it " +
-    "explicitly. See their documentation for details.", since = "2.13.0")
+  @migration(
+    "  The new ordering does not affect the sorting, placing NaN at the end.\n" +
+    "  However, methods such as `lt`, `min`, and `equiv` now match `compare`\n" +
+    "  instead of giving IEEE 754 behavior for -0.0F and NaN.\n" +
```

Comment:
I'd say "instead of having" rather than "instead of giving"

Reply:
acf7044